test/utils/refinement_test.py

```diff
@@ -78,6 +78,18 @@ def test_edge_volume(self):
         for low,high in zip(knots, knots[::-1]):
             self.assertAlmostEqual(low, 1-high)
 
+    def test_center_volume(self):
+        vol = Volume()
+        vol.raise_order(1,2,3)
+        edge_refine(vol, 1.3, 6, direction=2)
```

Comment:
Was the idea to test `center_refine` here?

Reply:
Yes it was 😊 Will fix and update PR